contracts/consensus/Consensus.sol

```diff
@@ -948,11 +1082,14 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
      * @param _metachainId Metachain id of a proposed metablock.
      * @param _dislocation Hash to shuffle validators.
      * @param _metablockHash Proposal under consideration for committee.
+     *
+     * @return committee_ CommitteeI instance for committee address
```

Comment:
This function is not returning anything. This documentation is not needed.

Reply:
We will need the committee address for emitting the event from `formCommittee` function as suggested from the below change, so updating the signature and the documentation is updated too